projects/cashmere/src/lib/pop/popover-anchoring.service.ts

```diff
@@ -252,9 +267,8 @@ export class HcPopoverAnchoringService implements OnDestroy {
                 case NotificationAction.TOGGLE:
                     this.togglePopover();
                     break;
-                case NotificationAction.REPOSITION:
-                    break;
                 // TODO: When the overlay's position can be dynamically changed, do not destroy
+                case NotificationAction.REPOSITION:
```

Comment:
Curious what this change was needed for

Reply:
Sorry I mashed together multiple tickets in this PR.  That fix you highlighted was for #1647.  The only way for those positioning changes to take effect is to make sure the tooltip gets destroyed and reinitialized when a position change happens.

Reply:
Ah got it! Thanks